lib/user.rb

```diff
@@ -0,0 +1,24 @@
+require 'pg'
+require_relative 'connection'
+require_relative '../database_connection.rb'
+
+class User
+
+  def self.create(email:, password:, display_name:)
+
+    result = Connection.query("INSERT INTO users (email, password, display_name) VALUES ('#{email}', '#{password}', '#{display_name}') RETURNING id, email, password, display_name;")
+    User.new(id: result[0]['id'],
+      email: result[0]['email'],
+      password: result[0]['password'],
+      display_name: result[0]['display_name']
+    )
+  end
+    attr_reader :id, :email, :password, :display_name
```

Comment:
Should we put this line at the tope just underneath class User to follow convention?